client/package.json

```diff
@@ -27,11 +28,12 @@
     "eslint-config-standard": "17.1.0",
     "eslint-plugin-nuxt": "4.0.0",
     "eslint-plugin-vue": "9.24.0",
-    "nuxt": "3.11.1",
+    "nuxt": "3.12.3",
```

Comment:
Anything significant in this upgrade that would warrant making it a separate PR?

Reply:
This was TS-related .. see https://github.com/nuxt/nuxt/discussions/25530#discussioncomment-9951962

Reply:
I'm happy to split up this PR though if you'd like

Reply:
No, seems on-topic enough